mitodl/open-discussions
# Added report counts to frontpage

#### What are the relevant tickets?
Fixes #495 

#### What's this PR do?
Adds the report counts to the frontpage and removes no longer needed calls to fetch the reports.

#### How should this be manually tested?
Verify that the counts still show up and you don't see report counts for channels you're not a moderator of.
